src/shared/ents.h

```diff
@@ -114,13 +114,12 @@ struct physent : baseent                        // can be affected by physics
     uchar type;                                 // one of ENT_* above
     uchar collidetype;                          // one of COLLIDE_* above
 
-    physent() : speed(100), weight(100), radius(3), aboveeye(1),
-        xradius(3), yradius(3), zradius(14), zmargin(0), curscale(1), speedscale(1),
+    physent() : speed(100), weight(100), radius(3), height(14), aboveeye(1),
```

Comment:
While we're at it, how about moving the initialization of all these numeric values from the constructor to the variable declaration? Also, I think this makes the initialization of `zradius` from `height` more robust, as the compiler will not just warn but error out if the order is mixed up (if `height` ever was declared first, its initial (uninitialized random) value will not be assigned to `zradius`, since it is unknown to the compiler at that stage).

Reply:
Wanted to do this but thought of just keeping the code the way it was, sure

Reply:
Valid point.